rpm/mic.spec

```diff
@@ -78,6 +78,7 @@ Requires:   yum >= 3.2.24
 %setup -q -n %{name}-%{version}
 
 %build
+sed 's/+.*//' <<<"%{version}" > VERSION
```

Comment:
Just by chance I came across the details and it seems it is correct to write the raw version there, as you were doing originally - see https://github.com/sailfishos/mic/blob/master/Makefile#L3

Reply:
Ah, yeah. I did not noticed that either, will fix still to original one.